WordPressAuthenticator/Extensions/UIPasteboard+Detect.swift

```diff
@@ -0,0 +1,24 @@
+extension UIPasteboard {
+
+    /// Detects patterns and values from the UIPasteboard. This will not trigger the pasteboard alert in iOS 14.
+    /// - Parameters:
+    ///   - patterns: The patterns to detect.
+    ///   - completion: Called with the patterns and values if any were detected, otherwise contains the errors from UIPasteboard.
+    @available(iOS 14.0, *)
+    func detect(patterns: Set<UIPasteboard.DetectionPattern>, completion: @escaping (Result<[UIPasteboard.DetectionPattern: Any], Error>) -> Void) {
+        UIPasteboard.general.detectPatterns(for: [.number]) { [weak self] result in
```

Comment:
Should we be using the passed in `patterns` value here, rather than hardcoding it for numbers? Same for the call to `detectValues` below.